src/components/Filter/components/Options/Options.tsx

```diff
@@ -0,0 +1,60 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+
+import useTheme from 'hooks/useTheme';
+import { generateTestDataId } from 'utils/helpers';
+import { FilterOption } from '../../types';
+import { defaultOptionStyle, optionStyle } from '../../utils';
+
+import { optionsStyle, emptyStyle } from './Options.style';
+
+interface Props {
+  items: FilterOption[];
+  onSelect: (option: FilterOption) => void;
+  defaultValue: FilterOption,
+  selectedItem?: FilterOption,
+  shouldDisplayDefaultOption: boolean;
+  dataTestId?: string;
+}
+const Options = ({
+   items,
+   onSelect,
+   defaultValue,
+   selectedItem,
+   shouldDisplayDefaultOption,
+   dataTestId,
+ }: Props) => {
+  const theme = useTheme();
+
+  return (
+    <div css={optionsStyle()(theme)}>
+      {shouldDisplayDefaultOption && <button
+        type="button"
+
+        data-testid={generateTestDataId('default-option', dataTestId)}
+        css={defaultOptionStyle(defaultValue, selectedItem)(theme)}
+        onClick={() => {onSelect(defaultValue);}}
```

Comment:
OCD kicking in! Could you remove the brackets here or add new lines? 

![image](https://user-images.githubusercontent.com/28539607/123055705-d732e680-d40e-11eb-9bd6-4a21b014e06b.png)

Reply:
yes you are right thank you